src/dataset.js

```diff
@@ -30,13 +30,16 @@ var Table = require('./table.js');
  * @class
  * @param {BigQuery} bigQuery {@link BigQuery} instance.
  * @param {string} id The ID of the Dataset.
+ * @param {object} [metadata] Dataset metadata.
+ * @param {string} [metadata.location] The geographic location of the dataset.
+ *      Required except for US and EU.
```

Comment:
Can you replace the this line with "Defaults to US."?

Reply:
Done!